### ISSUE TITLE
[@capacitor/google-maps] Add marker - option for clustering

### ISSUE THREAD

**Issue by AE1NS** [ionic-team/capacitor-plugins#1606]
### Plugin
@capacitor/google-maps

### Description
Currently there is only one cluster, that can be enabled or disabled. Its not possible to exclude markers form the clustering.

Example: There could be a map with multiple bus stops that should be clustered on zoom out. Additionally there could be the buses itself with moving positions, that should not be added to the cluster.

### Platform(s)
Android
iOS

### Preferred Solution
There could be an option for the addMarker method, where the developer can decide to add the marker to the cluster.

**Comment by RobSchilderr** [comment]
Yes! This is really necessary if you want to display markers and also the user's location. You don't want the user their location to become clustered too.

**Comment by ngmiduc** [comment]
Yes this would be very helpful !
